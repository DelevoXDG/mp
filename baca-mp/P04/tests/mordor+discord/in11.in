1 8
InsertFirst a b
InsertLast a b
Display a
Trains
Reverse a
Union a b
DelFirst a b
DelLast a b