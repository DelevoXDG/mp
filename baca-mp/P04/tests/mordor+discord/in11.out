Train a does not exist
Train a does not exist
Train a does not exist
Trains:
Train a does not exist
Train a does not exist
Train a does not exist
Train a does not exist